### DISCUSSION TITLE
What is the connection code to lego boost???

### DISCUSSION THREAD

**Discussion by ben7147** [pybricks/support#1100] [Q&A]
![image](https://github.com/pybricks/support/assets/95097279/d30153f4-f696-400b-9fa4-f84ec3f54ddb)

**Comment by ben7147** [comment]
.

**Reply by dlech** [reply]
Pybricks uses Bluetooth Low Energy. There is no passkey because the hubs don't connect this way. Only use Pybricks Code to connect to the hub.

**Reply by ben7147** [reply]
But I want to connect from VS code

**Reply by ben7147** [reply]
The code is PIN xd

**Reply by dlech** [reply]
If you want to use VS code, you can use the `pybricksdev` command line tool.
